### DISCUSSION TITLE
Benchmark

### DISCUSSION THREAD

**Discussion by hizaco** [carbon-language/carbon-lang#2021] [Getting started] [answered]
Hello Carboners,

Is there any existing benchmark for performances C++ vs carbon for the time being?

**Reverse-complement**

**Binary-trees**

**Fasta**

**N-body**

Thank you in advance

**Comment by jonmeow** [comment, marked as answer]
It's way too early for benchmarking: Carbon is not ready for use. https://github.com/carbon-language/carbon-lang#project-status

**Reply by hizaco** [reply]
Thank you :)
